content/en/monitoring/notification-manager.md

```diff
@@ -390,7 +390,7 @@ spec:
 EOF
 ```
 
-Slack receiver labeled with `type: tenant` only receive notification happened in the namespace which user be in. If you want to receive notification without namespace, you can create a global slack receiver labeled with `type: global` like this.
+The slack receiver labeled with `type: tenant` only receives notifications that happen in the namespace where the tenant has already existed. If you want them to receive notifications regardless of the namespace, you can create a global slack receiver labeled with type: global as follows:
```

Comment:
=>
Slack receivers with label `type: tenant` can only receive notifications from the namespace to which the specified tenant user has access. If you want them to receive notifications from all namespaces or even without a namespace label, you can create a global slack receiver with label `type: global` as below: